src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -76,9 +78,18 @@ public void Configure(TelemetryConfiguration configuration)
             if (this.telemetryProcessorFactories.Any())
             {
                 foreach (ITelemetryProcessorFactory processorFactory in this.telemetryProcessorFactories)
-                {                    
+                {
                     configuration.DefaultTelemetrySink.TelemetryProcessorChainBuilder.Use(processorFactory.Create);
-                }                
+                }
+            }
+
+            // Add any TelemetryInitializers added using DependencyInjection.
+            if (this.telemetryInitializerFactories.Any())
```

Comment:
I realize this is copied from the pattern for telemetryProcessorFactories immedialy above, but the `.Any()` check is pointless because, of course, the `foreach` handles it naturally. Remove it (and the one on line 76).